integration/configs/README.mkd

```diff
@@ -1,7 +0,0 @@
-Config
```

Comment:
@zreichert could you verify this `integration/configs` was the folder we discussed should be removed?  Also should these be removed as well: https://github.com/puppetlabs/r10k/tree/master/integration/scripts?  Thanks!